integration/mediation-tests/tests-patches/src/test/resources/artifacts/ESB/server/conf/axis2/axis2.xml

```diff
@@ -397,8 +397,8 @@
     <!--Uncomment this local transport to use local transport in mediation flow-->
     <!--<transportSender name="local" class="org.apache.axis2.transport.local.NonBlockingLocalTransportSender"/>-->
 
-    <!-- uncomment this and configure to use connection pools for sending messages>
-      <transportSender name="jms" class="org.apache.axis2.transport.jms.JMSSender"/-->
+    <!-- uncomment this and configure to use connection pools for sending messages-->
+    <transportSender name="jms" class="org.apache.axis2.transport.jms.JMSSender"/>
```

Comment:
Do we need this line anymore since we have now uncommented the test cases?

Reply:
Yes that is needed to run the ESBJAVA4279_MPRetryUponResponseSC_500_withNonRetryStatusCodes_200_and_202_TestCase testcase